src/gwt/src/org/rstudio/studio/client/workbench/views/source/editors/text/ace/ace-uncompressed.js

```diff
@@ -9696,81 +9696,83 @@ oop.inherits(Mode, CMode);
 exports.Mode = Mode;
```

Comment:
I don't think we need any changes to Ace at all to add a mode override as we are doing here.